charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,176 @@
+containerRuntime: docker
+Kubernetes: false
+
+operator:
+  initcontainer:
+    repository: "docker"
+    tag: "20.10"
+  container:
+    repository: "kubesphere/fluent-operator"
+    tag: "v1.7.0"
+    # FluentBit operator resources. Usually user needn't to adjust these.
+  resources:
+    limits:
+      cpu: 100m
+      memory: 60Mi
+    requests:
+      cpu: 100m
+      memory: 20Mi
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  logPath:
+    crio: /var/log
+    containerd: /var/log
+
+fluentbit:
+  image:
+    repository: "kubesphere/fluent-bit"
+    tag: "v1.9.9"
+  # fluentbit resources. If you do want to specify resources, adjust them as necessary
+  # You can adjust it based on the log volume.
+  resources:
+    limits:
+      cpu: 500m
+      memory: 200Mi
+    requests:
+      cpu: 10m
+      memory: 25Mi
+  # Specify custom annotations to be added to each FluentBit pod.
+  annotations: {}
+    # prometheus.io/scrape: "true"
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  secrets: []
+  # List of volumes that can be mounted by containers belonging to the pod.
+  volumes: []
+  # Pod volumes to mount into the container's filesystem.
+  volumesMounts: []
+
+  # Remove the above empty volumes and volumesMounts, and then set volumes and volumesMounts as below if you want to collect node exporter metrics
+# volumes:
+#   - name: hostProc
+#     hostPath:
+#       path: /proc/
+#   - name: hostSys
+#     hostPath:
+#       path: /sys/
+# volumesMounts:
+#   - mountPath: /host/sys
+#     mountPropagation: HostToContainer
+#     name: hostSys
+#     readOnly: true
+#   - mountPath: /host/proc
+#     mountPropagation: HostToContainer
+#     name: hostProc
+#     readOnly: true
+
+  # Set a limit of memory that Tail plugin can use when appending data to the Engine.
+  # If the limit is reach, it will be paused; when the data is flushed it resumes.
+  # if the inbound traffic is less than 2.4Mbps, setting memBufLimit to 5MB is enough
+  # if the inbound traffic is less than 4.0Mbps, setting memBufLimit to 10MB is enough
+  # if the inbound traffic is less than 13.64Mbps, setting memBufLimit to 50MB is enough
+  input:
+    tail:
+      memBufLimit: 5MB
+    # uncomment below nodeExporterMetrics section if you want to collect node exporter metrics
+#   nodeExporterMetrics:
+#     tag: node_metrics
+#     scrapeInterval: 15s
+#     path:
+#       procfs: /host/proc
+#       sysfs: /host/sys
+
+  # Configure the output plugin parameter in FluentBit.
+  # You can set enable to true to output logs to the specified location.
+  output:
+    es:
```

Comment:
I wonder if Kafka and ES is a good default - would just stdout be better so it always works or should we add it as a default as well?

Reply:
Because the output address needs to be customized, it is up to the user to decide. If it is not set here, it will take the default.

Reply:
Ah I see, you're disabling ES output which was not obvious to me. Having actual sections should configure things rather than disable them. 

Are you saying that if we don't disable it then it will always set up an ES output? That seems counter-intuitive to me.

Reply:
Configuring stdout as the default output has the following problems:
- If not excluding fluentbit pod's log, then its logs will be collected and output repeatedly
- If excluding fluentbit pod's log, then user will miss the important fluentbit error or warning to help them locate the root cause of fluentbit issues.

Having es and kafka output sections there but disabled is a kind of user-friendly default.
But we can add one comment here:
`# You can find more supported output plugins here: https://github.com/fluent/fluent-operator/tree/master/docs/plugins/fluentbit/clusteroutput`

This will also address the concern of Kafka default params 
![image](https://user-images.githubusercontent.com/18525465/211973050-ec7c665f-7b92-40c2-aaec-91d184342e5a.png)